docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+    api-dummy-fpm:
```

Comment:
our current api-dummy ended up using the built-in web server because of the ease of use (only one image to specify, no configuration); an nginx and fpm setup is more realistic, but a dummy is not necessarily realistic?

Reply:
The skeleton doesn't need caching etc, but multiple requests would become synchronous, making the Journal appear slow.

It's not that much extra to add here too.

Reply:
Skeleton-wise I think it's very good to have both nginx and fpm, since it's supposed to be touching roughly all pieces